frontend/src/metabase/admin/permissions/pages/GroupDataPermissionsPage/GroupsPermissionsPage.jsx

```diff
@@ -177,6 +182,11 @@ function GroupsPermissionsPage({
           onChange={handlePermissionChange}
           onAction={handleAction}
           onBreadcrumbsItemSelect={handleBreadcrumbsItemSelect}
+          messages={() => (
+            <>
+              {showSplitPermsMessage && <PermissionsEditorSplitPermsMessage />}
+            </>
+          )}
           warnings={() => (
```

Comment:
!nit messages and warnings being in different places feels weird. i'm wondering if we should rename the `messages` and `warnings` props to `preHeaderContent` and `postHeaderContent`.

Reply:
i still think this comment is relevant

Reply:
Fair enough. I'll get it adjusted lol